types/react-syntax-highlighter/react-syntax-highlighter-tests.tsx

```diff
@@ -74,6 +77,7 @@ function primsLightHighlighter(): JSX.Element {
     }
 }
 `;
+    PrismLightHighlighter.supportedLanguages; // $ExpectType string[]
```

Comment:
Since `supportedLanguages` is not available on prism-light, I believe this line should be removed.